wiki/Article_styling_criteria/Writing/th.md

```diff
@@ -0,0 +1,336 @@
+
+ชื่อผู้ใช้ที่มี `osu!` ซึ่งเป็นส่วนหนึ่งของชื่อหัวข้อจะต้องเป็นตัวพิมพ์ใหญ่ ตัวอย่างได้แก่
+
+- `osu! Alumni`
+- `osu! Champion`
+
+สำหรับเงื่อนไขอื่นๆ ทั้งหมด osu! ต้องถือว่าเป็น[คำนามที่มีคุณสมบัติ](https://en.wikipedia.org/wiki/Noun_adjunct) นี่หมายถึงการเพิ่มช่องว่างระหว่าง osu! และคำนามที่ปรับเปลี่ยน ตัวอย่างได้แก่
+
+- `osu! tournaments`
+- `osu! community`
+- `osu! chat`
+- `osu! client`
+- `osu! wiki`
+
+---
+
+`osu!` ไม่ควรจบประโยค หากใช่ ให้ใช้จุดเต็ม (`.`) หรือเครื่องหมายคำถามเพื่อรับ `osu!.` หรือ `osu!?`
+
+`osu!` ต้องไม่ตามด้วยเครื่องหมายอัศเจรีย์ (เช่น `osu!!`)   ประโยคจะต้องถูกเขียนใหม่เพื่อให้แน่ใจว่าสิ่งนี้จะไม่เกิดขึ้น
+
+## คำศัพท์
+
+คำเหล่านี้ต้องสะกดดังนี้ (การวรรคต้องตรงกัน):
+
+- `approach circle`
+- `game mode` (or `mode`)
+- `game mod` (or `mod`)
+- `gameplay`
+- `hit burst`
+- `hit circle`
+- `hitsound`
+- `in-game`
+- `playstyle`
+- `slider tick`
+- `slider ball`
+- `slider path`
+- `Kudosu` (เพื่อเป็นคำนามเฉพาะ)
+
+---
+
+บางคำมีรูปแบบต่างๆ ต้องใช้การสะกดคำที่ต้องการและมีดังนี้:
+
+- `creator` หรือ `mapper` แทน `beatmapper`.
+- `mapped` แทน `beatmapped`.
+- `BN` หรือ `Beatmap Nominators` เมื่อพูดถึง *ผู้เสนอบีทแมป*.
+- `sign in` แทน `log in`, เว้นแต่ชื่อของปุ่มหรือลิงก์จะระบุเป็นอย่างอื่น
+- `sign out` แทน `log out`, เว้นแต่ชื่อของปุ่มหรือลิงก์จะระบุเป็นอย่างอื่น
+- `register` แทน `sign up`, เว้นแต่ชื่อของปุ่มหรือลิงก์จะระบุเป็นอย่างอื่น
+
+### บทความการแข่งขัน
+
+คำเหล่านี้ต้องสะกดดังนี้ (การวรรคต้องตรงกัน):
+
+- `NoMods`, `NoMod`, หรือ `NM`
+- `FreeMods`, `FreeMod`, หรือ `FM`
+- `Tiebreaker`
+
+## ตัวย่อ, คำย่อ และอักษรย่อ
+
+ตัวย่อ, คำย่อ และอักษรย่อ ต้องมีความหมายเขียนออกมาเมื่อเกิดขึ้นครั้งแรก ครั้งอื่น ๆ เป็นทางเลือก แต่จะทำเมื่อจำเป็นเท่านั้น ต่อไปนี้เป็นตัวอย่าง:
+
+```markdown
+The NC (Nightcore) mod is similar to the DT (Double Time) mod because both NC and DT increase the speed of the music by 50%. However, NC will change the pitch of the music and add a clap and finish to the beat.
+```
+
+ตัวย่อ, คำย่อ และอักษรย่อ ต้องเป็นตัวพิมพ์ใหญ่ ยกเว้น ต่อไปนี้เป็นตัวอย่าง:
+
+- `CS` สำหรับ `Circle Size`
+- `AR` สำหรับ `Approach Rate`
+- `DT` สำหรับ `Double Time`
+- `SBS` สำหรับ `Storyboard Scripting`
+
+---
+
+ใช้ `e.g.` สำหรับ "ตัวอย่าง" และ `i.e.` สำหรับ "นั้นคือ".
+
+---
+
+ตัวย่อ, คำย่อ และอักษรย่อ ไม่ควรเป็นพหูพจน์ ต่อไปนี้เป็นตัวอย่าง:
+
+```markdown
+Avoid:  BNs can nominate your beatmap.
+
+Prefer: Beatmap Nominators can nominate your beatmap.
+```
+
+## ทัศนคติ
+
+`player`, `user`, `skinner`, `storyboarder`, และ `creator` ควรใช้เมื่อกล่าวถึงผู้อ่านหรือบุคคลอื่น `they`, `them`, `their`, และ `theirs` อาจใช้เมื่อจำเป็น
+
+`you` และ `your` ควรหลีกเลี่ยง `I` จะต้องไม่ใช้ `we`, `he` หรือ `his`, และ `she` หรือ `her` จะต้องไม่ใช้ (ดูข้อกำหนดอื่น ๆ ด้านบน)
+
+---
+
+[บทความที่มีการเขียนอย่างไม่เป็นทางการ](#ทะเบียนภาษา) อาจละเว้นส่วนนี้ อย่างไรก็ตาม ควรหลีกเลี่ยง `I` ในย่อหน้า
+
+## ไวยากรณ์
+
+บทความควรใช้ [ไวยากรณ์แบบอเมริกันที่ง่ายกว่า](https://www.thepunctuationguide.com/british-versus-american-style.html)
```

Comment:
this.. well, you get what I mean. use local equivalents or, if there is no such issue in your language, just drop the section, ASC is one of the few articles that don't have to match the original 1:1

Reply:
um yeah I should use english one for that

Reply:
deleted